src/api/bookData.js

```diff
@@ -54,10 +54,14 @@ const updateBook = (bookObj) => new Promise((resolve, reject) => {
 });
 
 // DONE: FILTER BOOKS ON SALE
-const booksOnSale = () => new Promise((resolve, reject) => {
-  axios.get(`${dbUrl}/books/.json?orderBy="sale"&equalTo=true`)
-    .then((response) => resolve(Object.values(response.data)))
-    .catch((error) => reject(error));
+const booksOnSale = (uid) => new Promise((resolve, reject) => {
+  // axios.get(`${dbUrl}/books/.json?orderBy="uid"&equalTo="${uid}"&orderBy="sale"&equalTo=true`)
```

Comment:
We should remove the commented-out code. If you need to save it add it to the readme. ✨